tests/e2e/test_api.py

```diff
@@ -26,29 +27,24 @@
 
 SCORE_THRESHOLD = 0.4  # low score is better
 
-# Sample Question/Answer set
-# TODO: Load this from QnA json file.
-QUESTION1 = "what is kubernetes?"
-ANSWER1 = (
-    "Kubernetes is an open source container orchestration tool developed by Google. "
-    "It allows you to run and manage container-based workloads, making it easier to "
-    "deploy and scale applications in a cloud native way. With Kubernetes, "
-    "you can create clusters that span hosts across on-premise, public, private, or "
-    "hybrid clouds. It helps in sharing resources, orchestrating containers across "
-    "multiple hosts, installing new hardware configurations, running health checks "
-    "and self-healing applications, and scaling containerized applications."
-)
-QUESTION2 = "what is openshift virtualization?"
-ANSWER2 = (
-    "OpenShift Virtualization is an add-on to the Red Hat OpenShift Container Platform "
-    "that enables the running and management of virtual machine workloads alongside "
-    "container workloads. It introduces new objects into the OpenShift cluster using "
-    "Kubernetes custom resources to facilitate virtualization tasks such as creating and "
-    "managing Linux and Windows virtual machines, running pod and VM workloads together "
-    "in a cluster, connecting to VMs through various consoles and CLI tools, importing "
-    "and cloning existing VMs, managing network interface controllers and storage disks "
-    "attached to VMs, as well as live migrating VMs between nodes."
-)
+
+@pytest.fixture(scope="module")
+def qna_data():
+    """Load Question/answer pair for evaluation."""
+    with open("tests/test_data/question_answer_pair.json") as qna_f:
+        qa_pairs = json.load(qna_f)
+
+    # TODO: Use arguments
+    return qa_pairs["gpt"]["with_rag"]
+
+
+def get_eval_question_answer(qna_pair, qna_id):
```

Comment:
add an argument here so we can choose between getting a RAG Q/A pair, and a no-rag one, and be clear about which we're getting/using.

Reply:
> As application is already started when we call e2e test. Model, with & without RAG option are already decided.

yes but this function can be called by multiple tests, and those tests can be marked as rag or no-rag